pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -96,6 +104,16 @@ func NewGPUDevices(name string, node *v1.Node) *GPUDevices {
 		klog.V(4).Infoln("name=", nodedevices.Name, "val=", *val)
 	}
 
+	for _, val := range nodedevices.Device {
+		klog.V(3).Infoln("Nvidia Device registered name=", nodedevices.Name, "val=", *val)
```

Comment:
Here is duplicated with line104，and we can change to `klog.V(4).InfoS("Nvidia Device registered name","name", nodedevices.Name, "val", *val)`.